### PR TITLE
Include composite objects from external references when generating script file

### PR DESCRIPTION
**Pull request by dovic95**
Add an `IncludeCompositeObjects` option that allows to include objects from the referenced database packages when generating the script file

### CODE REVIEW

File: src/DacpacTool/PackageBuilder.cs [hunk 1]
```diff
@@ -401,7 +401,9 @@ public void GenerateCreateScript(FileInfo dacpacFile, string databaseName)
 
             using var package = DacPackage.Load(dacpacFile.FullName);
             using var file = File.Create(Path.Combine(dacpacFile.DirectoryName, scriptFileName));
-            DacServices.GenerateCreateScript(file, package, databaseName);
+
+            var options = new DacDeployOptions() { IncludeCompositeObjects = true };
```
Comment (jmezach) [inline src/DacpacTool/PackageBuilder.cs:405]: It looks like we're hardcoding this to be true now which might not be what everyone wants. Ideally we'd pass the value of the MSBuild property down to here and use that.

**Review by jmezach — CHANGES_REQUESTED** [review]
@dovic95 Thank you for your contribution. I just have one comment.

File: src/DacpacTool/PackageBuilder.cs (hunk 1 above)
Comment (dovic95) [reply src/DacpacTool/PackageBuilder.cs]: Yes, you're right. I changed this,

File: README.md [hunk 2]
```diff
@@ -462,7 +463,9 @@ With this enabled you'll find a SQL script with the name `<database-name>_Create
 The database name for the create script gets resolved in the following manner:
 1. `TargetDatabaseName`.
 1. Package name.
-> Note: the generated script also uses the resolved database name via a setvar command.
+> Note: 
+>- the generated script also uses the resolved database name via a setvar command.
+>- if `IncludeCompositeObjects` is true, the composite objects (tables, etc.) from external references are also included in the generated script
```
Comment (jmezach) [inline README.md:468]: Perhaps it is worth noting here that `IncludeCompositeObjects` also defaults to true (see [here](https://github.com/rr-wfm/MSBuild.Sdk.SqlProj/blob/master/src/MSBuild.Sdk.SqlProj/Sdk/Sdk.props#L217)).

File: src/DacpacTool/PackageBuilder.cs [hunk 3]
```diff
@@ -389,7 +389,7 @@ private List<int> ParseSuppressionList(string suppressionList)
             return result;
         }
 
-        public void GenerateCreateScript(FileInfo dacpacFile, string databaseName)
+        public void GenerateCreateScript(FileInfo dacpacFile, string databaseName, bool includeCompositeObjects = false)
```
Comment (jmezach) [inline src/DacpacTool/PackageBuilder.cs:392]: Is there a reason for defining a default here? This is only called from one location right?

**Review by jmezach — APPROVED** [review]
Just a couple of small comments

File: src/DacpacTool/PackageBuilder.cs (hunk 3 above)
Comment (dovic95) [reply src/DacpacTool/PackageBuilder.cs]: Removed as it is called from a unique place. The other calls are from unit tests

File: README.md (hunk 2 above)
Comment (dovic95) [reply README.md]: Done

**Comment by jmezach** [comment]
LGTM. @ErikEJ @jeffrosenberg Perhaps either one of you can take a look here as well?

File: src/DacpacTool/PackageBuilder.cs [hunk 4]
```diff
@@ -161,7 +161,7 @@ public void SaveToDisk(FileInfo outputFile, PackageOptions packageOptions = null
             EnsureModelCreated();
             EnsureModelValidated();
             EnsureMetadataCreated();
-
```
Comment (jeffrosenberg) [inline src/DacpacTool/PackageBuilder.cs:164]: Looks like there are a bunch of unnecessary whitespace changes. Not the end of the world, but it would be nice to fix these.

**Review by jeffrosenberg — APPROVED** [review]
I'd prefer that you fix the whitespace issues before merging, if you can do that relatively easily, since it confuses the commitment history and a future `git blame`. Code looks good, though, so I'm going to just make that a suggestion and approve.

File: src/DacpacTool/PackageBuilder.cs (hunk 4 above)
Comment (dovic95) [reply src/DacpacTool/PackageBuilder.cs]: Fixed. This was due to my Rider settings
